www/js/survey/multilabel/trip-confirm-services.js

```diff
@@ -57,15 +57,16 @@ angular.module('emission.survey.multilabel.services', ['ionic', 'emission.i18n.u
           console.log("Starting promise execution with ", inputParams);
           omPromises = ch.INPUTS.map((item) => ch.getOptionsAndMaps(item));
           console.log("Promise list ", omPromises);
-          Promise.all(omPromises).then((omObjList) =>
-              ch.INPUTS.forEach(function(item, index) {
-                  inputParams[item] = omObjList[index];
-              })).catch((err) => {
-                    Logger.displayError("Error while loading input params in "+ch.INPUTS, err)
-                    reject(err);
-              });
-              console.log("Read all inputParams, resolving with ", inputParams);
-              resolve(inputParams);
```

Comment:
I guess this is a good change; but I don't see what it fixes, and whether it is necessary after removing the dashboard completely on non-multilabel surveys.

Reply:
The old code looks misleading and I don't think it does what it is intended to do. If you fix the intentation/alignment:
```js
Promise.all(omPromises).then((omObjList) =>
    ch.INPUTS.forEach(function (item, index) {
        inputParams[item] = omObjList[index];
    })
).catch((err) => {
    Logger.displayError("Error while loading input params in " + ch.INPUTS, err)
    reject(err);
});
console.log("Read all inputParams, resolving with ", inputParams);
resolve(inputParams);
```
You see that `resolve()` is called regardless of what happens to the inner `omPromises`.

However, now that the dashboard is disabled for non-MULTILABEL configurations, it doesn't actually cause problems anymore.
If you want me to revert back to the old code, which has evidently stood the test of time thus far, I will.

Reply:
Yes, let's please revert to the old code. Before changing, I would want to look at the blame to verify that there was no reason for it. I do remember that the input params loading was tricky when I wrote it.

Reply:
reverted 094b677167cab8d6aacd6ae06609168b9401f505